karavan-app/src/main/resources/configuration/kubernetes/build.sh

```diff
@@ -13,7 +13,7 @@ git clone --depth 1 --branch $GIT_BRANCH $GIT_REPOSITORY $CODE_DIR
 
 cd $CODE_DIR/$PROJECT_ID
 
-jbang -Dcamel.jbang.version=$CAMEL_VERSION camel@apache/camel export --local-kamelet-dir=$KAMELETS_DIR
+jbang -Dcamel.jbang.version=$CAMEL_VERSION camel@apache/camel export --local-kamelet-dir=$KAMELETS_DIR --runtime=${CAMEL_JBANG_RUNTIME:-camel-main}
```

Comment:
The runtime has been fixed in Camel 4.8.0, and was a bug in 4.7.0 only

Reply:
@davsclaus , thanks for the reply!

I just built the main branch (commit a1b0c339b7f355cbab71012d2d0f436573ed9ea5) and got the same error without the --runtime parameter

![image](https://github.com/user-attachments/assets/e9c2a605-fa5f-48d2-976f-ef757c66baf6)


sorry, but where is this 4.8.0 version?

Reply:
Can't find it

![image](https://github.com/user-attachments/assets/4deea997-1714-43d5-a264-38d8e154ce03)


Sorry, am i forgetting something?

Reply:
its apache camel 4.8 not karavan.

wait for karavan to release a new version after camel 4.8 is released

Reply:
thanks for the answer, @davsclaus 
should I remove this change, then?

Reply:
This ENV var would allow to control the runtime of choice so its IMHO fine to keep.
But karavan also have docker build scripts so I think changes in this PR should also be looked at in Docker and do something there also.

Reply:
done :)